### PR TITLE
adds terraform based dns management for JB domains in cloudflare

### PR DESCRIPTION
**Pull request by ironicbadger**

### CODE REVIEW

**Comment by ironicbadger** [comment]
- [x] Need ideas on how to manage tfstate - Linode S3? 
- [x] Need ideas on Github actions to automatically apply these changes when a PR is merged to this repo.
- [x] Also need to add support for `cloudflare_page_rule` for a few directs.

File: terraform/dns/terraform.tfvars [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+jb_com_pagerules = {
+  "new-jb-com" = {
+    target        = "new.jupiterbroadcasting.com"
+    forwarded_url = "https://www.jupiterbroadcasting.com"
+    status_code   = "301"
+  }
+  "www-jb-com" = {
+    target        = "jupiterbroadcasting.com"
+    forwarded_url = "https://www.jupiterbroadcasting.com"
```
Comment (ironicbadger) [inline terraform/dns/terraform.tfvars]: @noblepayne could you please update this value to reflect the needful from [#381](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/issues/381).

Comment (noblepayne) [reply terraform/dns/terraform.tfvars]: Updated :+1:

**Comment by ironicbadger** [comment]
Merging so that I can test #11

**Pull request merged**